man/GGIR.Rd

```diff
@@ -1271,7 +1276,21 @@ GGIR(mode = 1:5,
 
       \item{cosinor}{
         Boolean (default = FALSE). Whether to apply the cosinor analysis from the ActCR package.}
-
+        
+      \item{part6CR}{
+        Boolean (default = FALSE) to indicate whether circadian rhythm analysis should be run by part 6.
+      }
+      \item{part6HCA}{
+        Boolean (default = FALSE) to indicate whether Household Co Analysis should 
+        be run by part 6.
+      }
+      \item{part6Window}{
+        Character vector with length two (default = c("start", "end")) to indicate
```

Comment:
So, if I want to analyse circadian rhythms from first wakeup to last wakeup, this argument should be `part6Window = c("W1", "W-1")`, is this correct? I think this definition would benefit from a couple of quick examples on how to use it.